src/components/widgets/WebWidget.tsx

```diff
@@ -0,0 +1,39 @@
+import { useNavigation } from '@react-navigation/core';
+import React, { useCallback } from 'react';
+
+import { Icon, normalize, texts } from '../../config';
+import { ScreenName, WidgetProps } from '../../types';
+import { Image } from '../Image';
+
+import { DefaultWidget } from './DefaultWidget';
+
+export const WebWidget = ({ text, additionalProps }: WidgetProps) => {
+  const navigation = useNavigation();
+
+  const onPress = useCallback(() => {
+    navigation.navigate(ScreenName.Web, {
+      webUrl: additionalProps?.webUrl,
+      title: additionalProps?.staticContentTitle ?? texts.widgets.web
```

Comment:
please order alphabetically as in other places where we navigate to web screens

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/932/commits/cde16d832aa43fb8764d2ec780e2eb2548548ff3